test/unit/infrastructure/adapters/GitCasStateCacheRootSetCoordinator.test.ts

```
@@ -0,0 +1,402 @@
+      treeOid: this.headOid,
+      entries: [...this.entries],
+    };
+  }
+
+  async mutate(
+    mutator: (
+      entries: ReadonlyArray<Readonly<RootSetEntry>>,
+    ) => Iterable<RootSetEntry> | Promise<Iterable<RootSetEntry>>,
+  ): Promise<RootSetMutationResult> {
+    this.events.push('root:prepare');
+    this.entries = Array.from(await mutator(this.entries));
```

ESLint

[error] Use spread syntax [...iterable] instead of Array.from(iterable) when no mapper function is provided

(e18e/prefer-spread-syntax)